ios/AppLovinMAXNativeAdView.mm

```diff
@@ -100,6 +395,48 @@ - (void)setAdUnitId:(NSString *)adUnitId
     [self.isAdUnitIdSet set: YES];
 }
 
+- (void)setStrLocalExtraParameters:(NSArray<NSDictionary *> *)strLocalExtraParameters
```

Comment:
Should not use `set` prefix if we are not overriding a setter.
```suggestion
- (void)updateLocalExtraParametersForStringLocalExtraParameters:(NSArray<NSDictionary *> *)strLocalExtraParameters
```

Reply:
React Native generates the signature of `setStrLocalExtraParameters` since `strLocalExtraParameters` is a property of `<NativeAdView/>`.